### ISSUE TITLE
Feedback about ConvTranspose3d

### ISSUE THREAD

**Issue by mihribanyvs** [pytorch/pytorch#172560]
There is the following issue on this page: https://docs.pytorch.org/docs/stable/generated/torch.nn.ConvTranspose3d.html

I am currently using the ConvTranpose3d and also ConvTranspose2d function but the output shapes are not matching with the ones on the documentation. 

This is my input shape: torch.Size([1, 64, 128, 3]).
I am using a kernel size of (4,4) and a striding of (4,4), padding 0 and no dilation.
The output I get is torch.Size([1, 4, 256, 512]).

According to your documentation the output size should have been (128-1) x 4 - 0 + 1 = 509.
Can you correct the formula for the output shape in the documentation? Thank you.

cc @svekars @sekyondaMeta @AlannaBurke @albanD @mruberry @jbschlosser @walterddr @mikaylagawarecki
